docs/css/main.css

```diff
@@ -0,0 +1,521 @@
+  padding: 16px 0;
+  text-align: left;
+}
+
+.main-item > img {
+  width: auto;
+  height: 100px;
+  padding-right: 32px;
+}
+
+.main-item h2 {
+  margin-top: 0;
+}
+
+.compatibility {
+  background-color: #F2F2F2;
+}
+
+.browsers > img {
+  width: 70px;
+  height: 70px;
+  padding: 20px;
+}
+
+footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 14px;
+  background: #F2F3F4;
+  color: rgba(0,0,0,.4);
+  border-top: 1px solid #dadce0;
+}
+
+footer ul {
+  padding: 0;
+}
+
+footer ul li {
+  display: inline-block;
+}
+
+footer ul li::before {
+  padding: 0 14px;
+  content: '/';
+  color: rgba(0,0,0,.4);
+}
+footer a {
+  color: #202124;
+  font-size: 14px;
+}
+
+/* doc page */
+.toc {
+  font-family: 'Poppins', 'Roboto', 'Noto', sans-serif;
+  /* position: sticky works in Chrome, Firefox, Safari and Edge;
+  fallback to position: static for which the nav will just be part
+  of the scrolling content. */
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  left: 0;
+  width: 340px;
+  height: 300px;
+  box-sizing: border-box;
+  padding: 24px;
+}
+
+.toc a:hover {
+  border-bottom: 2px solid #FADE4B;
+  -webkit-transition: -webkit-transform .3s ease;
+  -webkit-transition: transform .3s ease;
+  transition: transform .3s ease;
+}
+
+.toc ul {
+  padding: 0 0 0 16px;
+}
+
+.toc ul ul {
+  padding: 8px 0 0 40px;
+}
+
+.toc li {
+  padding: 6px 0;
+  list-style-type: none;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 1.5;
+  -webkit-transition: -webkit-transform .3s ease;
+  -webkit-transition: transform .3s ease;
+  transition: transform .3s ease;
+}
+
+.toc > ul > li {
+  font-weight: bold;
+}
+.toc svg {
+  width: 24px;
+  height: 24px;
+  fill: currentcolor;
+  vertical-align: middle;
+}
+.toc span {
+  display: inline-block;
+  vertical-align: sub;
+  padding: 0 10px;
+}
+
+article {
+  max-width: 860px;
+  padding: 40px;
+  margin: -300px 0 0 340px;
+  border-left: 1px solid #dadce0;
+}
+
+article h1 {
+  font-size: 30px;
+  font-weight: 600;
+  letter-spacing: normal;
+  line-height: 1.1;
+  margin: 0 0 24px 0;
+  padding-bottom: 8px;
+}
+article h1:after {
+  display: block;
+  width: 60px;
+  height: 5px;
+  content: ' ';
+  background: #FADE4B;
+  margin: 20px 0;
+}
+
+article h2 {
+  font-weight: 600;
+  margin: 64px 0 24px;
+  padding-bottom: 8px;
+}
+
+article h3 {
+  font-weight: 600;
+  margin: 32px 0 16px;
+}
+
+article a {
+  color: #1e88e5;
+}
+
+article a:hover {
+  text-decoration: underline;
+}
+
+article img {
+  max-width: 600px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 4px 0 rgba(0,0,0,0.14);
+  margin: 20px auto;
+  display: block;
+}
+ h2 img {
+   display: inline-block;
+   vertical-align: middle;
+ }
+
+p.caption, b {
+  font-weight: 500;
+}
+
+code, pre {
+  font-family: 'Source Code Pro', 'Roboto Mono', Consolas, Menlo, monospace !important;
+}
+
+code {
+  background: #F4F4F6;
+  color: black;
+  font-size: inherit;
+  padding: 4px 8px;
```

Comment:
The padding makes the code block looks weird. Can we remove it?
<img width="629" alt="screen shot 2018-06-12 at 4 34 24 pm" src="https://user-images.githubusercontent.com/116360/41322653-f8c9dc72-6e5e-11e8-87f4-fb1382111980.png">